xclim/sdba/_adjustment.py

```diff
@@ -161,8 +163,8 @@ def mbcn_train(
     Parameters
     ----------
     Dataset variables:
-        ref : training target
-        hist : training data
+        ref : training target, DataArray (stacked variables)
+        hist : training source, DataArray (stacked variables)
```

Comment:
This should open with:
```
ds : xr.Dataset
    Dataset variables:
        ...

Reply:
ok, I will change all docstrings in _adjustment, that's not specific to my PR

Reply:
Hmm, I get "Unexpected indentation." for 
```python
    ds : xr.Dataset
        Dataset variables:
        ref : training target
        hist : training source
```
or
```python
    ds : xr.Dataset
        Dataset variables:
            ref : training target
            hist : training source
```
any other suggestion?

Reply:
I guess we can't use ":" in the descriptions. What about:
```
ds : xr.Dataset
    Dataset variables must be :
    ref (training target)
    hist (training source)
```

Reply:
got it, I had to fight with my tabs